.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=sqlite:///home/redempt/.tabby/ee/db.sqlite
```

Comment:
This is temporary. We will need to either:

1. Have people update this file with their local path to the tabby DB
2. Use the build.rs to automatically generate this file with the correct path
3. Include a SQLite db in the tabby repository storing only the schema

This is for sqlx's checked queries.